.changeset/smooth-monkeys-fetch.md

```diff
@@ -0,0 +1,6 @@
+---
+'curvenote': patch
```

Comment:
Ugh, meant for this to be minor, but I guess I hit `enter` instead of `space` in the cli.

But I don't think it maters since there is another minor bump in this changeset.

Reply:
Yep, we can fix it up and ensure that these are correct when we bump.